RichClipboard.package/RCClippingsWindow.class/instance/initialize.st

```diff
@@ -4,7 +4,11 @@ initialize
 	self updateWindowTitle.
 	self
 		clippingsList: (RCClippingsList newWithStorage: self storage).
+	self privateModeActive: false.
 	self addClippingsList.
+	(self privateModeActive) ifFalse: [ self clippingsList canCopy ].
 	self addToolbar.
 	self addToToolbar: self createRegexSearchButton.
-	self addToToolbar: self createPrivateModeButton
\ No newline at end of file
+	self addToToolbar: self createPrivateModeButton
+	
+	
```

Comment:
Bitte statt "self privateModeActive: false" und "(self privateModeActive) ifFalse: [ self clippingsList canCopy ]." 

"self turnOffPrivateMode" benutzen und in turnOffPrivateMode clippingsList darüber informieren, dass private mode aus ist (und bitte wie bereits gesagt canCopy umbenennen)